src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperContent.cs

```diff
@@ -202,6 +202,13 @@ public TagHelperContent SetContent(IHtmlContent htmlContent)
         /// <returns>A <see cref="string"/> containing the content.</returns>
         public abstract string GetContent();
 
+        /// <summary>
+        /// Gets the content.
+        /// </summary>
+        /// <param name="encoder">The <see cref="IHtmlEncoder"/>.</param>
+        /// <returns>A <see cref="string"/> containing the content.</returns>
+        public abstract string GetContent(IHtmlEncoder encoder);
```

Comment:
Though I can't imagine this to be a big extensibility point of tag helpers, isn't it icky to have *two* abstract methods that do essentially the same thing -- especially when one calls the other? I think a more traditional pattern would be for the parameterless method to be virtual and call the parameterful method with the default `HtmlEncoder`, no? (And then don't override the parameterless method in the derived type.)